src/platform/BUILD.gn

```diff
@@ -113,6 +112,10 @@ if (chip_device_platform != "none") {
       defines += [ "CHIP_DEVICE_CONFIG_ENABLE_CHIPOBLE=${chip_enable_ble}" ]
     }
 
+    if (chip_enable_sed) {
```

Comment:
Do we need a separate GN argument at all? Most apps seem to set CHIP_DEVICE_CONFIG_ENABLE_SED via CHIPProjectConfig.h and we could do the mapping from Kconfig in platform/nrfconnect/CHIPDevicePlatformConfig.h. I think a common rule is that GN arguments are added for bigger settings that require you to modify which files and how are to be compiled.

Reply:
Removed as suggested